sunpy/visualization/draw.py

```diff
@@ -0,0 +1,163 @@
+
+    Returns
+    -------
+    visible : `~matplotlib.patches.Polygon`
+        The patch added to the axes for the visible part of the solar equator.
+    hidden : `~matplotlib.patches.Polygon`
+        The patch added to the axes for the hidden part of the solar equator.
+
+    Examples
+    --------
+    >>> import sunpy.map
+    >>> from sunpy.visualization import draw
+    >>> import sunpy.data.sample   # doctest: +REMOTE_DATA
+    >>> aia = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)   # doctest: +REMOTE_DATA
+    >>> fig = plt.figure()   # doctest: +SKIP
+    >>> ax = fig.add_subplot(projection=aia)   # doctest: +SKIP
+    >>> aia.plot()   # doctest: +SKIP
+    >>> draw.equator(ax)   # doctest: +SKIP
+    """
+    if not wcsaxes_compat.is_wcsaxes(axes):
+        raise ValueError('axes must be a WCSAxes')
+    axes_frame = wcsapi_to_celestial_frame(axes.wcs)
+
+    lat = 0*u.deg
+    lat0 = SkyCoord(np.linspace(-180, 179, resolution) * u.deg,
+                    np.ones(resolution) * lat, radius=rsun,
+                    frame=HeliographicStonyhurst(obstime=axes_frame.obstime))
+    visible, hidden = _plot_vertices(lat0, axes, axes_frame, rsun,
+                                     **kwargs)
+    return visible, hidden
+
+
+def prime_meridian(axes, rsun: u.m = R_sun, resolution=500, **kwargs):
+    """
+    Draws the solar prime meridian as seen by the axes observer.
+    Hidden parts are drawn as a dotted line.
+
+    Parameters
+    ----------
+    axes : `matplotlib.axes.Axes`
+        The axes to plot the prime meridian on, or "None" to use current axes.
+    rsun : `~astropy.units.Quantity`
+        Solar radius (in physical length units) at which to draw the solar
+        prime meridian. Defaults to the standard photospheric radius.
+    resolution : `int`
+        The number of points used to represent the prime meridian.
+
+    Returns
+    -------
+    visible : `~matplotlib.patches.Polygon`
+        The patch added to the axes for the visible part of the solar equator.
+    hidden : `~matplotlib.patches.Polygon`
+        The patch added to the axes for the hidden part of the solar equator.
+    """
+    if not wcsaxes_compat.is_wcsaxes(axes):
+        raise ValueError('axes must be a WCSAxes')
+    axes_frame = wcsapi_to_celestial_frame(axes.wcs)
+
+    if not hasattr(axes_frame, 'observer'):
+        raise AttributeError('an observer is required on the WCSAxes')
+    observer = axes_frame.observer
+
+    lon = 0*u.deg
+    lon0 = SkyCoord(np.ones(resolution) * lon,
+                    np.linspace(-90, 90, resolution) * u.deg, radius=rsun,
+                    frame=HeliographicCarrington(observer=observer,
+                                                 obstime=axes_frame.obstime))
+    visible, hidden = _plot_vertices(lon0, axes, axes_frame, rsun,
+                                     pm_dis=True, **kwargs)
+    return visible, hidden
+
+
+def _plot_vertices(coord, axes, frame, rsun, pm_dis=False, **kwargs):
```

Comment:
```suggestion
def _plot_vertices(coord, axes, frame, rsun, close_path=True, **kwargs):
```
Rename this keyword to be more general